backend/chess/rewards.py

```diff
@@ -1,4 +1,5 @@
 MOVE = -1
+UPGRADE_PIECE = 1

```

Comment:
willen we niet meer rewards hieraan geven? dis wel een beetje een groot doel van eht spel

Reply:
ah ok zal ik hem nu zo merge of wou je eerst die test afwachetn